src/components/onboarding/OnBoardingModal/OnBoardingContentContainer.tsx

```diff
@@ -116,6 +128,17 @@ export default function OnBoardingContentContainer({
             />
           </MutedDiv>
         )}
+
+        {customContinueButtonGroup &&
+          customContinueButtonGroup({
```

Comment:
Why is it function? Maybe do it as component?

Reply:
Last review with @teodorus-nathaniel , we want to move related states into `Signer.tsx`.

I have refactored the code, now as component in 1d61787